c++/src/kj/thread.h

```diff
@@ -59,7 +59,7 @@ class Thread {
     Function<void()> func;
     kj::Maybe<kj::Exception> exception;
 
-    int refcount;
+    unsigned int refcount;
```

Comment:
Confused here -- the MSDN docs say that `_InterlockedDecrement()` takes a signed value, which is why I made this signed. Were they lying?

BTW KJ defines `uint` as an abbreviation.